website/src/components/Metadata.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import Helmet from "react-helmet";
+
+function Metadata() {
+  return (
+    <Helmet>
+      {/*<!-- HTML Meta Tags -->*/}
+      <title>Bedav - Find hospital beds for COVID Patients</title>
+      <meta name="description" content="Find hospital bed availability for COVID Patients across India. Currently available for major cities in Karnataka, Maharashtra, Andhra Pradesh."/>
+      {/*<!-- Google / Search Engine Tags -->*/}
+      <meta itemProp="name" content="Bedav - Find hospital beds for COVID Patients"/>
+      <meta itemProp="description" content="Find hospital bed availability for COVID Patients across India. Currently available for major cities in Karnataka, Maharashtra, Andhra Pradesh."/>
+      <meta itemProp="image" content="https://www.1-more-thing.com/wp-content/uploads/2020/04/modified.png"/>
+      {/*<!-- Facebook Meta Tags -->*/}
+      <meta property="og:url" content="https://bedav.org"/>
+      <meta property="og:type" content="website"/>
+      <meta property="og:title" content="Bedav - Find hospital beds for COVID Patients"/>
+      <meta property="og:description" content="Find hospital bed availability for COVID Patients across India. Currently available for major cities in Karnataka, Maharashtra, Andhra Pradesh."/>
+      <meta property="og:image" content="https://www.1-more-thing.com/wp-content/uploads/2020/04/modified.png"/>
+      {/*}<!-- Twitter Meta Tags -->*/}
+      <meta name="twitter:card" content="summary_large_image"/>
+      <meta name="twitter:title" content="Bedav - Find hospital beds for COVID Patients"/>
+      <meta name="twitter:description" content="Find hospital bed availability for COVID Patients across India. Currently available for major cities in Karnataka, Maharashtra, Andhra Pradesh."/>
+      <meta name="twitter:image" content="https://www.1-more-thing.com/wp-content/uploads/2020/04/modified.png"/>
```

Comment:
I got it online.

Reply:
>     * Is it possible to attach a screenshot of the updated website (both on mobile and desktop)?
Desktop:
![image](https://user-images.githubusercontent.com/58643011/97749743-d084e700-1aac-11eb-9a86-104b6efcb6f0.png)
Mobile:
![image](https://user-images.githubusercontent.com/58643011/97749775-df6b9980-1aac-11eb-83d9-1defd7d2d54a.png)
>     * Also, I noticed you're using `react-helmet` to attach the meta tags the `head` tag of the page. Is there any advantage of doing this with `react-helmet` rather than directly putting the meta tags into the header?
When the site JS takes a longer time to load, the website indexers usually skip the site. If the meta tags are added with server side render, they will help the search engines get relevant information without waiting for it to load.

Reply:
Currently, the initial render times are around 2 seconds, so I think we should be good. But yes server side rendering will definitely help with both SEO and performance for the end user.